proxy/src/console/messages.rs

```diff
@@ -28,6 +28,7 @@ impl fmt::Debug for GetRoleSecret {
 pub struct WakeCompute {
     pub address: Box<str>,
     pub aux: MetricsAuxInfo,
+    pub allowed_ips: Option<Vec<Box<str>>>,
```

Comment:
I noticed that we store this responses to the cache.
Would it make sense to cache allowed_ips as well?

Reply:
Ok, I see, that ttl is quite small, so it's fine.

Reply:
Hmm. Do we want this on wake_compute or on get_auth_info

Reply:
Hm, I think, the original suggestion was to add it to `wake_compute` (https://github.com/neondatabase/neon/issues/5628#issuecomment-1781420603).

But, probably, you are right and get_auth_info would make more sense.